VideoLocker/res/values/styles.xml

```diff
@@ -109,6 +109,10 @@
         <item name="android:background">@color/edx_grayscale_neutral_light</item>
     </style>
 
+    <style name="light_grey_separator" parent="separator_base">
+        <item name="android:background">@color/edx_grayscale_neutral_xx_light</item>
```

Comment:
We should use consistent spelling here - since we use "gray" in `edx_grayscale_neutral_xx_light`, we should use the same spelling in here as well (also in the base `grey_separator`).